.ci/update-github-milestone.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+  VAR_NAME=$1
+  if [ ! -v "$VAR_NAME" ]; then
+    echo "Error: Define $1 environment variable"
+    exit 1
+  fi
+}
+
+checkForVariable "READ_ONLY_TOKEN"
+
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name" | sed 's/checkstyle-//' | sed 's/"//' | sed 's/"//')
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+echo "Updating milestone at github"
+MILESTONE_ID=$(curl -s \
+                -X GET https://api.github.com/repos/checkstyle/checkstyle/milestones?state=open \
+                H "Authorization: token $READ_ONLY_TOKEN" \
+                | jq ".[0] | .number")
+curl \
+  -X PATCH \
+  -H "Authorization: token $READ_ONLY_TOKEN" \
+  https://api.github.com/repos/checkstyle/checkstyle/milestones/"$MILESTONE_ID" \
+  -d "{ \"title\": \"$LATEST_RELEASE_TAG\", \
+        \"state\": \"open\", \
```

Comment:
you do not need to set "state", keep only items to change.

Reply:
Changes Done